.buildkite/deploy/kubernetes/deploy-k8s-pipeline.yml

```diff
@@ -23,6 +27,10 @@ steps:
     commands:
       - "MODULE=kubernetes make -C metricbeat integration-tests"
       - "make -C deploy/kubernetes test"
+    if: build.env("BUILDKITE_PULL_REQUEST") != "false" && build.env("GITHUB_PR_LABELS") =~ /.*kubernetes.*/
+    notify:
```

Comment:
I'd remove notifications from each step. One top-level notification would be enough